flocker/provision/_install.py

```diff
@@ -122,6 +129,17 @@ def task_upgrade_kernel():
     ]
 
 
+def task_upgrade_kernel_centos():
+    return [
+        Run.from_args([
+            "yum", "install", "-y", "kernel-devel", "kernel"]),
+        # For dkms and ... ?
+        Run.from_args([
+            "yum", "install", "-y", "epel-release"]),
```

Comment:
I wonder how people will feel about installing this repository and the software it provides.  Presumably they're running CentOS 7 because they like it more than Fedora.  I could imagine there being some push back here, maybe?

I just talked to Luke about this and he seems to think installing from EPEL is cool though.  So nothing to worry about here.